lib/jsonapi/serializable/conditional_fields.rb

```diff
@@ -0,0 +1,65 @@
+module JSONAPI
+  module Serializable
+    module ConditionalFields
+      def self.prepended(klass)
+        klass.class_eval do
+          extend ClassMethods
+          prepend InstanceMethods
+          class << klass
```

Comment:
I think `class << self` would be a little less confusing, as we're within a `class_eval` already here.